conda_build/main_build.py

```diff
@@ -30,6 +30,7 @@
     'numpy': [16, 17, 18, 19, 110],
     'perl': None,
     'R': None,
+    'lua': ["2.0", "5.1", "5.2", "5.3"]
```

Comment:
>perl and R both allow . characters in their versions.

Oops, I meant to include a link:
https://github.com/conda/conda-build/blob/master/conda_build/config.py#L21